crypto/src/main/kotlin/net/corda/v5/crypto/merkle/IndexedMerkleLeaf.kt

```diff
@@ -2,6 +2,16 @@ package net.corda.v5.crypto.merkle
 
 import net.corda.v5.base.annotations.CordaSerializable
 
+/**
+ * [IndexedMerkleLeaf]s are building blocks of [MerkleProof]s.
+ * They contain the required information about a particular leaf which is needed for the verification.
+ *
+ * @property index The leaf's index.
+ * @property nonce The leaf's nonce.
```

Comment:
perhaps say: nonce the leaf's optional nonce.

Reply:
Thanks. I have updated it.